### PR TITLE
Feature/har 125 styleguide

### PR DESCRIPTION
**Pull request by romypetitjean**
To do: 
- Create space around headings in content component (h1 up to h6)

### CODE REVIEW

File: scss/components/content.scss [hunk 1]
```diff
@@ -0,0 +1,221 @@
+$content-a-border: ($spacing / 8) solid; // NOTE: No color is set to inherit the color of its parent
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: Still not convinced we should use spacing for borders.

File: scss/components/content.scss [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+	& > :first-child {
+		margin-top: 0;
+	}
+
+	& > :last-child {
+		margin-bottom: 0;
+	}
+}
+
+.content--diap {
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: Content dark?

File: scss/components/content.scss [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+	color: $color-text-diap;
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: color-text-light?

File: scss/components/content.scss [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+}
+
+h1 {
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: Why not  '.content h1'? .content is not a state, modifier or pseudo-class/element. Or nest everything in one .content.

File: scss/components/content.scss [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+			font-size: $font-size-100;
+		}
+	}
+}
+
+ol,
+ul {
+
+	.content & {
+		max-width: 40em;
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: Should this be set for li?

File: scss/components/content.scss [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+		line-height: $line-height-100;
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: Should this be set for li?

File: scss/components/content.scss [hunk 7]
```diff
@@ -0,0 +1,221 @@
...
+
+		@include media(0, $range-hand) {
+			margin-top: $line-height-80 * $font-size-100;
+			margin-bottom: $line-height-80 * $font-size-100;
+			padding-left: $line-height-80 * $font-size-100;
+			font-size: $font-size-100;
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: Should this be set for li?

File: scss/components/content.scss [hunk 8]
```diff
@@ -0,0 +1,221 @@
...
+	.content & {
+		list-style: disc;
+	}
+}
+
+a {
+
+	.content & {
+
+		&:not(.button) {
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: Is this even possible?

File: scss/components/content.scss [hunk 9]
```diff
@@ -0,0 +1,221 @@
...
+			border-bottom: $content-a-border;
+		}
+	}
+}
+
+strong {
+
+	.content & {
+		font-weight: bold;
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: bold or 600? H1 is 600, strong is bold.

File: scss/components/headings.scss [hunk 10]
```diff
@@ -0,0 +1,140 @@
+.headings {
```
Comment (bartvandebiezen) [inline scss/components/headings.scss]: Prefere heading-group

File: scss/components/headings.scss [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+	& > *:first-child {
+		margin-top: 0;
+	}
+
+	& > *:last-child {
+		margin-bottom: 0;
+	}
+}
+
+.headings--diap {
```
Comment (bartvandebiezen) [inline scss/components/headings.scss]: --dark?

File: scss/components/headings.scss [hunk 12]
```diff
@@ -0,0 +1,140 @@
...
+		color: $color-text-diap;
+	}
+}
+
+.headings__overline {}
+
+.headings__headline {
+	color: $color-heading-200;
+	font-family: $font-family-display;
+	font-weight: 700;
```
Comment (bartvandebiezen) [inline scss/components/headings.scss]: 700 or bold? Not every font has 700.

File: scss/components/headings.scss [hunk 13]
```diff
@@ -0,0 +1,140 @@
...
+	}
+
+	.headings--diap & {
+		color: $color-text-diap;
+	}
+}
+
+.headings__title {
+	color: $color-heading-200;
+	font-weight: 400;
```
Comment (bartvandebiezen) [inline scss/components/headings.scss]: I think we should set this bold for harbour.

File: scss/components/headings.scss [hunk 14]
```diff
@@ -0,0 +1,140 @@
...
+	}
+
+	.headings--diap & {
+		color: $color-text-diap;
+	}
+}
+
+.headings__subtitle {
+	color: $color-heading-200;
+	font-weight: 400;
```
Comment (bartvandebiezen) [inline scss/components/headings.scss]: normal or 400?

File: scss/components/headings.scss [hunk 15]
```diff
@@ -0,0 +1,140 @@
+.headings {
+
+	& > *:first-child {
+		margin-top: 0;
```
Comment (bartvandebiezen) [inline scss/components/headings.scss]: margin-top and margin-bottom are missing from elements.

File: scss/components/headings.scss [hunk 16]
```diff
@@ -0,0 +1,140 @@
...
+		font-size: $font-size-200;
+	}
+
+	.headings--diap & {
+		color: $color-text-diap;
+	}
+}
+
+.headings__emphasis {
+	color: $color-ui-accent;
```
Comment (bartvandebiezen) [inline scss/components/headings.scss]: color-text-accent?

File: scss/components/menu.scss [hunk 17]
```diff
@@ -0,0 +1,78 @@
+$menu-item-vertical-padding: $spacing * 1.5;
+$menu-padding-item-horizontal-padding: $spacing * 2;
+$menu-inline-item-horizontal-padding: $spacing;
+
+.menu {}
```
Comment (bartvandebiezen) [inline scss/components/menu.scss]: I think we should discuss this more before we add it to harbour.

File: scss/layouts/collection.scss [hunk 18]
```diff
@@ -0,0 +1,92 @@
+.collection {
```
Comment (bartvandebiezen) [inline scss/layouts/collection.scss]: I think we should discuss this more before we add it to harbour.

File: scss/layouts/page.scss [hunk 19]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+.page__main {
+	display: flex;
+	flex-direction: column;
+	flex-grow: 1;
+	flex-shrink: 0;
+}
+
+.page__main--background-colored {
```
Comment (bartvandebiezen) [inline scss/layouts/page.scss]: just --colored?

File: site/_layouts/docs.html [hunk 20]
```diff
@@ -0,0 +1,51 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		{% include head.html %}
+	</head>
+	<body>
+		<section class="page">
```
Comment (bartvandebiezen) [inline site/_layouts/docs.html]: body should be 'page'?

File: site/_layouts/docs.html [hunk 21]
```diff
@@ -0,0 +1,51 @@
...
+			<header class="page__header">
+				{% include page-header.html %}
+			</header>
+			<main class="page__main">
+				<div class="layout">
+						<div class="layout__section layout__section--bordered-bottom">
```
Comment (bartvandebiezen) [inline site/_layouts/docs.html]: section element?

File: site/_layouts/docs.html [hunk 22]
```diff
@@ -0,0 +1,51 @@
...
+							<div class="layout__inner layout__inner--padded-90">
+								<div class="layout__columns">
+									<div class="layout__column layout__column--two-thirds">
+										<div class="headings">
```
Comment (bartvandebiezen) [inline site/_layouts/docs.html]: header element?

File: scss/components/content.scss (hunk 4 above)
Comment (bartvandebiezen) [inline scss/components/content.scss]: missing margins

File: scss/components/content.scss [hunk 23]
```diff
@@ -0,0 +1,221 @@
...
+		@include media($range-hand) {
+			font-size: $font-size-500;
+		}
+		@include media($range-lap, false) {
+			font-size: $font-size-600;
+		}
+	}
+}
+
+h2 {
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: missing margins

File: scss/components/content.scss [hunk 24]
```diff
@@ -0,0 +1,221 @@
...
+		@include media($range-hand) {
+			font-size: $font-size-400;
+		}
+		@include media($range-lap, false) {
+			font-size: $font-size-500;
+		}
+	}
+}
+
+h3 {
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: Add margins

File: scss/components/content.scss [hunk 25]
```diff
@@ -0,0 +1,221 @@
...
+		@include media($range-hand) {
+			font-size: $font-size-300;
+		}
+		@include media($range-lap, false) {
+			font-size: $font-size-400;
+		}
+	}
+}
+
+h4 {
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: Add margins

File: scss/components/content.scss [hunk 26]
```diff
@@ -0,0 +1,221 @@
...
+		@include media($range-hand) {
+			font-size: $font-size-200;
+		}
+		@include media($range-lap, false) {
+			font-size: $font-size-300;
+		}
+	}
+}
+
+h5 {
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: Add margins

File: scss/components/content.scss [hunk 27]
```diff
@@ -0,0 +1,221 @@
...
+		@include media(0, $range-hand) {
+			font-size: $font-size-100;
+		}
+		@include media($range-lap, false) {
+			font-size: $font-size-200;
+		}
+	}
+}
+
+h6 {
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: Add margins

File: scss/components/content.scss [hunk 28]
```diff
@@ -0,0 +1,221 @@
...
+}
+
+p {
+
+	.content & {
+		max-width: 40em;
+		line-height: $line-height-100;
+
+		@include media(0, $range-hand) {
+			margin-top: $line-height-80 * $font-size-100;
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: line-height-100

File: scss/components/content.scss [hunk 29]
```diff
@@ -0,0 +1,221 @@
...
+			margin-bottom: $line-height-80 * $font-size-100;
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: line-height-100

File: scss/components/content.scss [hunk 30]
```diff
@@ -0,0 +1,221 @@
...
+			font-size: $font-size-100;
+		}
+		@include media($range-lap, false) {
+			margin-top: $line-height-80 * $font-size-200;
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: line-height-100

File: scss/components/content.scss [hunk 31]
```diff
@@ -0,0 +1,221 @@
...
+			margin-bottom: $line-height-80 * $font-size-200;
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: line-height-100

File: scss/components/content.scss [hunk 32]
```diff
@@ -0,0 +1,221 @@
...
+			font-size: $font-size-200;
+		}
+	}
+}
+
+p {
+
+	.content & {
+		max-width: 40em;
+		line-height: $line-height-100;
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: Should this be line height 100? Should we change line-height-100 from 1.5 to 1.25?

File: scss/components/content.scss [hunk 33]
```diff
@@ -0,0 +1,221 @@
...
+			margin-top: $line-height-80 * $font-size-200;
+			margin-bottom: $line-height-80 * $font-size-200;
+			font-size: $font-size-200;
+		}
+	}
+
+	.content--90 & {
+
+		@include media(0, $range-hand) {
+			margin-top: $line-height-80 * $font-size-90;
```
Comment (bartvandebiezen) [inline scss/components/content.scss]: All margins should use same line-height as the line-height.

File: scss/components/headings.scss [hunk 34]
```diff
@@ -0,0 +1,140 @@
...
+.headings__underline {
+	color: $color-heading-30;
+	text-transform: uppercase;
+	line-height: $line-height-40;
+
+	@include media(0, $range-hand) {
+		font-size: $font-size-90;
+	}
+	@include media($range-lap) {
+		margin-top: $spacing / 4;
```
Comment (bartvandebiezen) [inline scss/components/headings.scss]: I think the margins should be set based on line-height and font-size. Possible a half of that.

File: scss/components/menu.scss [hunk 35]
```diff
@@ -0,0 +1,78 @@
...
+
+.menu--inline {
```
Comment (bartvandebiezen) [inline scss/components/menu.scss]: inlined or inlining?

File: scss/components/menu.scss [hunk 36]
```diff
@@ -0,0 +1,78 @@
...
+	display: flex;
```
Comment (bartvandebiezen) [inline scss/components/menu.scss]: Not wrapping?

File: scss/components/menu.scss (hunk 35 above)
Comment (bartvandebiezen) [reply scss/components/menu.scss]: Or --horizontal?

File: scss/components/menu.scss [hunk 37]
```diff
@@ -0,0 +1,78 @@
...
+	.menu--inline &,
+	.menu--inline-centered & {
+		padding-right: $menu-inline-item-horizontal-padding;
+		padding-left: $menu-inline-item-horizontal-padding;
+		width: auto;
+		border-top: none;
+	}
+
+	.menu--padding & {
+		padding-right: $menu-padding-item-horizontal-padding;
```
Comment (bartvandebiezen) [inline scss/components/menu.scss]: When do you use menu--padding?

File: scss/components/menu.scss [hunk 38]
```diff
@@ -0,0 +1,78 @@
...
+		padding-left: $menu-padding-item-horizontal-padding;
+	}
+
+	// .menu--forward-icon & {
```
Comment (bartvandebiezen) [inline scss/components/menu.scss]: Commented code.

File: scss/layouts/page.scss [hunk 39]
```diff
@@ -0,0 +1,30 @@
+.page {
+	display: flex;
+	flex-direction: column;
+	min-height: 100%;
+
+	@include is-ie-10-or-11 {
```
Comment (bartvandebiezen) [inline scss/layouts/page.scss]: Works better:
```
@include media-ie-10-11 {
	display: table;
	width: 100%;
	height: 100%;
}
```

File: scss/layouts/page.scss [hunk 40]
```diff
@@ -0,0 +1,30 @@
...
+		height: 100%; // FIX For IE11 use 'height' instead of 'min-height'. The height will increase because 'page__main' has 'flex-grow: 1;' which overrules height of parent;
+	}
+}
+
+.page__header {
```
Comment (bartvandebiezen) [inline scss/layouts/page.scss]:
```
@include media-ie-10-11 {
	display: table-row;
	height: 1px;
}
```

File: scss/layouts/page.scss [hunk 41]
```diff
@@ -0,0 +1,30 @@
...
+	flex-grow: 0;
+	flex-shrink: 0;
+}
+
+.page__main {
```
Comment (bartvandebiezen) [inline scss/layouts/page.scss]:
```
@include media-ie-10-11 {
	padding-bottom: $spacing * 0.5;
	height: 100%;
}
```

File: scss/layouts/page.scss [hunk 42]
```diff
@@ -0,0 +1,30 @@
...
+	flex-direction: column;
+	flex-grow: 1;
+	flex-shrink: 0;
+}
+
+.page__main--background-colored {
+	background-color: $color-background-colored;
+}
+
+.page__footer {
```
Comment (bartvandebiezen) [inline scss/layouts/page.scss]:
```
@include media-ie-10-11 {
	display: table-row;
	height: 1px;
}
```

File: scss/layouts/toolbar.scss [hunk 43]
```diff
@@ -0,0 +1,73 @@
+.toolbar {
```
Comment (bartvandebiezen) [inline scss/layouts/toolbar.scss]: I think we should discuss this more before we add it to harbour.

File: scss/settings/colors.scss [hunk 44]
```diff
@@ -21,8 +21,8 @@ $color-primary: $color-harbour-blue !default;
 $color-secondary: $color-harbour-purple !default;
 
 // ### Typography
-$color-heading: $color-harbour-blue !default;
-$color-subheading: $color-harbour-blue !default;
+$color-heading: $color-harbour-black !default;
```
Comment (bartvandebiezen) [inline scss/settings/colors.scss]: dark?

File: site/_layouts/home.html [hunk 45]
```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		{% include head.html %}
+	</head>
+	<body>
```
Comment (bartvandebiezen) [inline site/_layouts/home.html]: class page?

File: site/_layouts/home.html [hunk 46]
```diff
@@ -0,0 +1,27 @@
...
+		<section class="page">
+			<header class="page__header">
+				{% include page-header.html %}
+			</header>
+			<main class="page__main">
+				<div class="layout">
+					<div class="layout__section">
+						<div class="layout__inner">
+							<div class="content">
```
Comment (bartvandebiezen) [inline site/_layouts/home.html]: section element

File: scss/components/content.scss (hunk 9 above)
Comment (stevengunneweg) [reply scss/components/content.scss]: should this not use a font-weight variable?

File: scss/components/content.scss (hunk 7 above)
Comment (michaelmaat) [reply scss/components/content.scss]: Discussed: Bart will fix `ul` and `li` margins

File: scss/components/content.scss (hunk 9 above)
Comment (michaelmaat) [reply scss/components/content.scss]: Use bold for now

**Pull request merged**